home/templates/home/tags/banners_list.html

```diff
@@ -2,8 +2,12 @@
 
 <section class='banner-holder'>
     {% for banner in banners %}
-        <a href="{% pageurl banner.banner_link_page %}">
-            {% image banner.banner_image width-360 %}
-        </a>
+        {% if banner.banner_link_page == None %}
+            <a>{% image banner.banner_image width-360 %}</a>
```

Comment:
could you try using span instead of a tag as it shouldn't be clickable without linked page